src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -129,11 +136,12 @@ class SubjectHeading extends React.Component {
     return `${path}/subject_headings/${uuid}`;
   }
 
-  updateSort(sortType, direction) {
+  updateSort(sortType, direction, numberOpen) {
```

Comment:
Maybe we should call this `updateNestedSort`, because doing a global search in the code for `updateSort` returns a lot of results.

Reply:
The passing of this new prop is a little strange. Seems like there should be a better approach. But not a big deal.